### DISCUSSION TITLE
CoolProp Guid Excel Guide on Website out of Date: Where to find current .xlam files?

### DISCUSSION THREAD

**Discussion by raphaelagner** [CoolProp/CoolProp#2354] [Q&A]
Hi,

we're using CoolProp for teaching and wanted to update the installation instructions. The instructions for the Excel Wrapper in MacOS on the webiste (http://www.coolprop.org/coolprop/wrappers/Excel/index.html) are unfortunately leading to dead links.

Can anyone clarify where to find the .xlam file? i've found them up until 6.4.1. on sourceforge

Thanks!

**Comment by ibell** [comment]
Your best best is to run the windows installer, it installs the Excel add-in. Files are here: https://sourceforge.net/projects/coolprop/files/CoolProp/6.6.0/Installers/Windows/

**Comment by ibell** [comment]
I just checked and it seems to work for me.
